### PR TITLE
Fixes for Bug #1182497

### PR DESCRIPTION
**Pull request by kwwii**
1. Help loupe - QT 
2. Title font size - QT
3. Scrollbar colour - fixed
4. Checkbox - fixed
5. Horizontal frame - fixed
6. Text wrapping - QT
7. Row selection - fixed
8. Question mark icon - ALL icons in dialogues should have been removed a long time ago
9. Broken frame in software selection - fixed
10. Collapse/expand icon in software selection and system tasks - fixed
11. Font glyph sizes - won’t fix
12. Expand/collapse triangle in filesystem options - QT
13. Missing border in partition menu - it’s a different type of menu? Ideally, the white top/bottom of the other menus would not be dark grey but I cannot figure out how :-(
14. Calendar does not show days of the week - fixed
15. Overview errors not visible enough - won’t fix

### CODE REVIEW

**Comment by joseivanlopez** [comment]
It looks good to me in general. Some comments:

>     1. Help loupe - QT
For the time being, I would say this is a wontfix.

>     8. Question mark icon - ALL icons in dialogues should have been removed a long time ago
So, should this be fixed in YaST side?
 
>     13. Missing border in partition menu - it’s a different type of menu? Ideally, the white top/bottom of the other menus would not be dark grey but I cannot figure out how :-(
*QMenuBar* and *QMenu* classes allow to adapt styles for the menu. For example:

~~~
QMenu {
  border: 1px solid white;
}
~~~

Could you please try it?

**Comment by kwwii** [comment]
> It looks good to me in general. Some comments:
> 
> > ```
> > 1. Help loupe - QT
> > ```
> 
> For the time being, I would say this is a wontfix.

Agreed.


> > ```
> > 8. Question mark icon - ALL icons in dialogues should have been removed a long time ago
> > ```
> 
> So, should this be fixed in YaST side?

Yes, it should.


> 
> > ```
> > 13. Missing border in partition menu - it’s a different type of menu? Ideally, the white top/bottom of the other menus would not be dark grey but I cannot figure out how :-(
> > ```
> 
> _QMenuBar_ and _QMenu_ classes allow to adapt styles for the menu. For example:
> 
> ```
> QMenu {
>   border: 1px solid white;
> }
> ```
> 
> Could you please try it?

I would prefer to remove the light areas on the top and bottom of the other menus than make all of them exhibit the same incorrect design. In order to make the menu stand out better against the background, I will change the outer line a bit.

**Comment by kwwii** [comment]
I have changed the style of the QMenu to have more contrast to the background in my latest commit.

**Review by joseivanlopez — CHANGES_REQUESTED** [review]
Could you please add a changelog entry and bump the version? See for example https://github.com/yast/yast-theme/pull/132/commits/0bbd2b1cc9a54c36a223051703d5d7975541e987.

File: package/yast2-theme.changes [hunk 1]
```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Tue Mar  2 13:21:43 UTC 2021 - Kenneth Wimer <wimer@suse.com>
+
+- Fixes for bug #1182497
```
Comment (joseivanlopez) [inline package/yast2-theme.changes:4]: I think you have to use the bsc#XXXXX format, otherwise our bots will complain:

~~~
- Fixes for bsc#1182497
~~~

**Review by joseivanlopez — APPROVED** [review]
LGTM. Changelog will be fix with a separate PR.

**Comment by dgdavid** [comment]
> LGTM. Changelog will be fix with a separate PR.

See https://github.com/yast/yast-theme/pull/139

**Pull request merged**